### DISCUSSION TITLE
Suggestion to make Relax and Sudden Death/Perfect game modifications mutually inclusive.

### DISCUSSION THREAD

**Discussion by ArchUsr64** [ppy/osu#13229] [Ideas] [answered]
As of osu!lazer 2021.524.0, `Relax` mod is mutually exclusive to both `Sudden Death` and `Perfect` mod. 
As a player who plays exclusively offline with relax mod, it's annoying to have to manually retry a beatmap when going for a FC or SS. The ability to use Sudden Death and Perfect mod alongside Relax mod will be appreciated.

**Comment by goodtrailer** [comment]
What if Relax was changed to no longer block fails? I'd like the ability to fail normally while playing Relax, and the No Fail mod exists if you don't want that. Making Relax fail-able makes adding compatibility with Sudden Death/Perfect easier (it should be trivial, may be wrong though).

**Reply by Totsugeki1** [reply]
Yeah, I've been a (mostly) Relax player for the past one year and a half and I feel like all of that should (maybe) be a customizable part of the mod, or just a variation of it (like NC is to DT). This doesn't just apply to Relax, but Autopilot too.

**Reply by TheFantasticLoki** [reply]
I second these opinions as not being able to fail at all with relax was something I always didn't like. This was a huge part of custom clients on private servers that had relax mods independant of the built in relax mod which allowed you to fail, submit scores, and have combo count properly towards your score. While scores will be able to submit with relax in lazer and combo is properly accounted for (THANK YOU OMG) I still feel being able to fail is something that should be implemented.

**Reply by cdwcgt** [reply]
if relax is allow to fail, it need to reconsider the multiplier of relax.

But it's very simple to use it as an option.

**Reply by Asayaka** [reply]
What if relax/autopilot have multiplier 0,50 or 0,40 if it just plain relax (doesnt use no fail)

**Comment by peppy** [comment]
I'm not against changing the way relax behaves if there is majority support for it.

**Reply by trustytrojan** [reply]
Hope this gets implemented sooner or later

**Comment by Asayaka** [comment]
What if there is a menu to turn off Cant Fail or Turn On Fail in Relax. So people who want to have relax mode can fail can turn on that feature

**Reply by ghost** [reply]
IIRC currently mod compatibility cant be dependant on mod settings. With your suggestion either you'd be able to put perfect with fail turned off, or not be able to put on perfect at all.

**Reply by Asayaka** [reply]
What I said is just like the DT Speed change menu. It's just Optional. It's not like you must turn on Fail (If we have that feature) in Relax. **Relax is different than Perfect or SD** . Relax is just like AutoPilot; you can play the game by only moving your cursors. It doesn't like No Fail / SD Mod that have spesific task (Like if you miss you lost or You cant fail in NF) . So for me, it's okay to add some settings optional just like in HD mod where you can choose only fade approach circles feature.

**Comment by bdach** [comment, marked as answer]
This was changed in https://github.com/ppy/osu/pull/25917.
